src/sequelize.js

```diff
@@ -242,7 +242,7 @@ class Sequelize {
 
       // For postgres, we can use this helper to load certs directly from the
       // connection string.
-      if (['postgres', 'postgresql'].includes(options.dialect)) {
+      if (['postgres', 'postgresql', 'yugabyte'].includes(options.dialect)) {
```

Comment:
we should just use 'yugabytedb' or both(confirm that first if 'yugabyte' is allowed to use in this case)